CHANGELOG.md

```diff
@@ -17,4 +17,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## 0.4.1
 
-- Updated hyperlinks in README.md
\ No newline at end of file
+- Updated hyperlinks in README.md
+
+## 0.4.2
```

Comment:
This file is upside down. Can you clean it up to before a first release?

Reply:
The most recent versions should be on top, not at the bottom.

Reply:
done